test/units/testsuite-48.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -ex
+
+test_rule="/run/udev/rules.d/48-test.rules"
+
+setup() {
+    mkdir -p "${test_rule%/*}"
+    cp -f /etc/udev/udev.conf /etc/udev/udev.conf.bckp
+    echo 'KERNEL=="lo", SUBSYSTEM=="net", PROGRAM="/usr/bin/sleep 60"' > "${test_rule}"
```

Comment:
this needs to be /bin/sleep, not /usr/bin/sleep

Reply:
also just to be pedantic, PROGRAM is a match key, not an assignment key, so PROGRAM== is the correct way to specify it, but it doesn't really matter in this case as the important part is just running the program to force the event timeout, and anyway udev-rules assumes '==' when just '=' is specified.